lib/Importer/ONNXIFIModelLoader.cpp

```diff
@@ -197,6 +197,12 @@ ONNXIFIModelLoader::parseOperators(const void *onnxModel,
       ADD_OP_MAPPING(MinNodeKind, FloatTy);
       ADD_OP_MAPPING(MaxNodeKind, FloatTy);
       ADD_OP_MAPPING(SplatNodeKind, FloatTy);
+    } else if (operation == "BatchMatMul") {
+      ADD_OP_MAPPING(TransposeNodeKind, FloatTy);
```

Comment:
In some cases `BatchMatMul` is just one `MatMulNode`. In some cases there's just no transpose. How can we account for this?

Reply:
Currently the strategy is to cover the max-set which we support anyway. `BatchMatMul` is really a hack from ONNX point of view, maybe we can export it more appropriately, but this is not a priority for now.

Reply:
Tracking this is 1950 issue